tools/integration/test/integration/fixtures/definitions/conda-conda--forge-linux--aarch64-numpy-1.16.6--py36hdc1b780_0.json

```diff
@@ -5295,8 +5290,7 @@
         {
             "path": "lib/python3.6/site-packages/numpy/polynomial/chebyshev.py",
             "attributions": [
-                "(c) Covert Chebyshev",
-                "(c) Multiply a Chebyshev"
+                "(c) Covert Chebyshev"
```

Comment:
This change takes out several incorrect copyright detections but leaves in the detection of Covert Chebyshev on line 134 appears which to be incorrect as it is only referring to a function not a copyright. That is comments talking about Convert Chebyshev series to z-series..
Worth noting that there is a reference to a paper by Chebyshev, as shown below, on line 105, but this is not a copyright statement, it is a reference.
  Polynomials," *Journal of Statistical Planning and Inference 14*, 2008
  (https://web.archive.org/web/20080221202153/https://www.math.hmc.edu/~benjamin/papers/CombTrig.pdf, pg. 4)

This doesn't impact approval/rejection of PR as it merely moves a poor quality detection down a line, but just noting as looked into it.